### PR TITLE
Sign in

### PR DESCRIPTION
**Pull request by Rockyraven**
build sign page
add email and password input box
and login button
login page
n

### CODE REVIEW

File: Quickstore/pages-styles/signin.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    height: 2rem;
+    width: 15rem;
+    margin: 1rem 3.6rem;
+}
+
+.forget-password{
+    margin-left: 2rem;
+    text-decoration: none;
+}
+.content{
```
Comment (LordUttam) [inline Quickstore/pages-styles/signin.css]: If you are using this class somewhere have a descriptive name or just remove it.

File: Quickstore/pages-styles/signin.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    justify-content: center;
+    align-items: center; 
+}
+.login-container{
+    height: 25rem;
+    width: 22rem;
+    background-color: var(--default-color);
+    margin: 5rem;
+    text-align: center;
+
```
Comment (LordUttam) [inline Quickstore/pages-styles/signin.css]: remove these empty lines. Run prettier.

File: Quickstore/pages/signin.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <a href="#"><i class="social-icon fa fa-heart"></i><p>Wishlist</p></a>
+            <a href="#"><i class="social-icon fa fa-shopping-cart"></i><p>Cart</p></a>
+        </div>
+    </nav>
+
+    
+    <hr>
+           <div class="login-area">
+               <div class="login-container">
+                    <h2 >Login</h2>
```
Comment (LordUttam) [inline Quickstore/pages/signin.html:37]: Run prettier here too. File has unintentional empty lines and spaces.

**Review by LordUttam — COMMENTED** [review]
No major problems. Rest looks good. 👍

**Review by pratik-kumar-sahu — APPROVED** [review]
- Looks good to be merged. Please resolve above comments.

**Review by vishalg8454 — APPROVED** [review]
Can merge

**Pull request merged**